Data-Collection-Inventory/2021Q1_csvs/specimen_type.csv

```diff
@@ -1,14 +1,10 @@
 specimen_type_name
 brain hemisphere
-brain region
 brain section set
 brain slice
 bulk nucleus
-cell body
 cell in slice
 cell nucleus
-retina
-spinal cord
```

Comment:
keep; some data collections do have whole spinal cord. There may be an issue if this is mis-labeled on some of the data collections

Reply:
specifically zhangl_anatomy is whole spinal cord (not slices)

Reply:
specifically zhangl_anatomy is whole spinal cord (not slices)

Reply:
Changed `zhangl_anatomy` specimen type to (whole) `spinal cord` to fix this.